performance/implementations/glamor/Box/index.js

```diff
@@ -6,12 +6,7 @@ import View from '../View';
 const Box = ({ color, fixed = false, layout = 'column', outer = false, ...other }) => (
   <View
     {...other}
-    style={[
-      styles[`color${color}`],
-      fixed && styles.fixed,
-      layout === 'row' && styles.row,
-      outer && styles.outer
-    ]}
+    style={[styles[`color${color}`], fixed && styles.fixed, layout === 'row' && styles.row, outer && styles.outer]}
```

Comment:
I think we only bumped up to 120 at Twitter because our imports (especially with relative paths) can break over 2 or more lines with the 80 char limit. You might not experience that so much at FB with your module system :). OK so I might drop back to 80 for now since it's so easy to do!

Really impressed with the results so far - great work! This will be a useful case study to include in a proposal to adopt `prettier` at Twitter. Let me know if you have any recommendations / warnings related to doing that.